content/chainguard/chainguard-images/unique-tags/index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Unique Tags for Chainguard Images"
+linktitle: "Unique Tags"
+type: "article"
+description: "Overview of what Chainguard's Unique Tags are and how to access them."
+date: 2024-02-29T08:49:31+00:00
+lastmod: 2024-02-29T08:49:31+00:00
+draft: false
+tags: ["Chainguard Images", "Product", "Overview"]
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 735
+toc: true
+---
+
+Some organizations use image tags as an indication that there is a new container image available in a registry. Oftentimes, these organizations' internal automation and policies expect each new build to have its own distinct tag. 
+
+To help with cases like this, Chainguard offers Unique Tags for private registries. Unique Tags are ideal for organizations that require a strict tag per release or update of their images. They benefit teams looking for precise tracking and management of container images. 
+
+This guide provides an overview of what these Unique Tags are, the kinds of problems they aim to solve, and how you can access them in the Chainguard Console.
+
+
+## Chainguard's Unique Tags
+
+Unique Tags are only available for private registries, as the public Chainguard Registry only has the `:latest` or, in some cases, the `:latest-dev` tags available. Unique Tags feature an opt-in feature, which allows customers the flexibility to enable it based on their specific operational and security requirements.
+
+Chainguard's Unique Tags end in a timestamp, such as `20240229`, which indicates the date when the Image was built. Because Chainguard Images are rebuilt daily, this ensures that the specific tag will always represent that specific Image build and not another.
```

Comment:
I might change the wording around this, since there's been confusion in the past about whether everything is rebuild every day, or rebuilt when there are upstream changes to packages.

Something like this maybe?
"Because Chainguard Images are rebuilt whenever there is a change to an included package, the timestamp ensures that the specific tag will always represent that specific Image build and not another."